scripts/apps/templates/controllers/CreateTemplateController.ts

```diff
@@ -33,19 +33,20 @@ function applyMiddleware(_item: IArticle, content, vocabularies): Promise<IArtic
                     const config = vocabulary.custom_field_config ?? {};
                     const customField = fieldTypes[vocabulary.custom_field_type];
 
-                    itemNext = {
-                        ...itemNext,
-                        extra: {
-                            ...itemNext.extra,
-                            [fieldId]: customField.onTemplateCreate(
-                                itemNext?.extra?.[fieldId],
-                                config,
-                            ),
-                        },
-                    };
+                    if (!itemNext?.extra?.[fieldId] === null) {
```

Comment:
What was the issue and solution here? I would rather always call `onTemplateCreate` to give the extension the opportunity to set a default.

Reply:
suppose we have a template with the Coming up field active then if we create an article based on the template, switch the Coming Upfield off and use the 3 dots menu, do "save as template" and update the existing template. 
So the expected behavior is that the Coming Upfield should be disabled in the template. currently, if the coming upfield is disabled it always comes as enabled so to fix this issue.

Reply:
- suppose we have a template with the Coming up field active then if we create an article based on the template, switch the Coming Upfield off and use the 3 dots menu, do "save as template" and update the existing template. 
So the expected behavior is that the Coming Upfield should be disabled in the template. currently, if the coming upfield is disabled it always comes as enabled so to fix this issue. 
- and also current behavior is just the opposite of the ticket **Coming Upfield disappears from template after the template is updated by Save as template from authoring [SDBELGA-628]**

Reply:
I see. Could you instead modify date time extension to return `null`, but still call `onTemplateCreate` here?

Reply:
yep I will do that.